### DISCUSSION TITLE
Looking for L-Shaped Jarvis standup desk Parts

### DISCUSSION THREAD

**Discussion by ventu** [phord/Jarvis#31] [General]
So, as some may know, the Fully brand is being axed tomorrow and I am in need of parts. I picked up this Jarvia L-Shaped Standing desk at an Auction for $23 and it's missing some pieces.

The two Actuator cables to plug into the control box and the Memory Handset which can be replaced by the steps in the GItHub. Would anyone here have any Javris-related parts or cables for sale?
https://www.workwhilewalking.com/has-fully-com-gone-out-of-business

**Comment by OkhammahkO** [comment]
I think Jarvis is Jiecang OEM right?

So perhaps you can also look at other brands?

There's a Jiecang list here.

https://community.home-assistant.io/t/desky-standing-desk-esphome-works-with-desky-uplift-jiecang-assmann-others/383790?u=mahko_mahko

Or maybe check out the Jiecang site?
https://www.jiecang.com/Control-Boxes.html
